pkg/binutils/processes.go

```diff
@@ -64,6 +64,22 @@ func NewGRPCClient() (client.Client, error) {
 	if errors.Is(err, context.DeadlineExceeded) {
 		err = ErrGRPCTimeout
 	}
+	if client != nil {
+		ctx := GetAsyncContext()
+		rpcVersion, err := client.RPCVersion(ctx)
+		if err != nil {
+			return nil, err
+		}
+		// obtained using server API
+		serverVersion := rpcVersion.Version
+		// obtained from ANR source code
+		clientVersion := server.RPCVersion
+		if serverVersion != clientVersion {
+			return nil, fmt.Errorf("trying to connect to a backend controller that uses a different RPC version (%d) than the CLI client (%d). Use network clean to stop the controller and then restart the operation",
```

Comment:
hmm, do we want a way to do this that does not require clearing your deployed subnets? Maybe like a `network clean --server-only` flag that preserves your snapshot?

Reply:
mmm do you prefer  `network clean --server-only` (need to add snapshot preservation to clean) or a `network stop --shutdown-server` (will add server shutdown to stop)

Reply:
`network stop --shutdown-server` is better